array_methods_lab.js

```diff
@@ -0,0 +1,185 @@
+// console.log('-----QUESTION 1------')
+// /*iven an array of numbers, write a function that
+// returns a new array where every element is incremented by 10.*/
+//
+// let numArr = [1,2,3,4,5]
+// let addTen = numArr.map((crrElem) => {
+//   return crrElem +  10
+// })
+// console.log(addTen)
+
+
+
+
+// console.log('********QUESTION 2*********')
+// /*Given an array, return a new array where every
+// non-string element is converted to an empty string.*/
+//
+// let mixedArr = ["a", 123, "b", "c", {name: "cat"}]
+// let strArr = mixedArr.map((crrElem) => {
+//   if (typeof(mixedArr) === 'number') {
+//   return ""
+//   }
+//   return mixedArr
+// })
+// console.log(strArr)
+
+
+
+
+// console.log('((((((((((((QUESTION 3))))))))))))')
+// //Write a function that returns only the even elements in an array of numbers.
+//
+// let ranArr = [1,2,3,4,5]
+// let evenArr = ranArr.filter((crrElem) => {
+//   if (crrElem % 2 === 0) {
+//     return true
+//   }
+//   false
+// })
+// console.log(evenArr)
+
```

Comment:
Good job.. We have to remember the use of our array methods. filter, filters out whatever doesn't comply with our condition. So return true or false is not necessary. we can just return our condition.

let evenArr = ranArr.filter((crrElem) => {
	 return (crrElem % 2 === 0) 
 })